src/client/app/components/HeaderComponent.jsx

```diff
@@ -38,8 +39,9 @@ export default function HeaderComponent(props) {
 				<h1 style={titleStyle}>Open Energy Dashboard</h1>
 			</div>
 			<div style={divRightStyle}>
-				<Link style={loginLinkStyle} to="/login"><button className="btn btn-default">Log in</button></Link>
-				<Link style={adminLinkStyle} to="/admin"><button className="btn btn-default">Admin panel</button></Link>
+				{(props.renderOptionsButton && props.drawUIOptionsButton) ? <UIModalComponent /> : null}
```

Comment:
That makes sense, but I'm not sure where else to put it. Don't forget that on mobile (where this is important), you really don't want things to the left or right of the chart, you want it above or below. This places it above with all the other buttons.